viewer/viewer.js

```diff
@@ -212,16 +212,16 @@ if (Config.get('passwordSecret')) {
     }
 
     // S2S Auth
-    if (req.headers['x-moloch-auth']) {
-      const obj = Config.auth2obj(req.headers['x-moloch-auth'], false);
+    if (req.headers['x-arkime-auth']) {
```

Comment:
Can you make this accept both the x-moloch-auth and x-arkime-auth, same with line below

Reply:
👍